### ISSUE TITLE
Does this project is only to replace tail -F command?

### ISSUE THREAD

**Issue by zuoruochen** [alexwangx/flume-ng-source#1]
I am confused with this source, if it's only to replace tail -F command. In fact, i think a tail source should record the offset of a tailed file periodically to keep the reliability. I want to know whether this source have done this?

**Comment by alexwangx** [comment]
Yes, this project consists of two functions, one is the tail directory, and the other is to achieve unix platform above the tail -F command.
Because unix platform above the tail no -F option. Therefore clone the flume-og the source to achieve a simple tail -F command.
Here does not implement data reliability.
I have been a long time did not participate in this flume needs. If you implement this function, welcome to submit here.
-:)
